Raven.NET.Core/Subjects/RavenSubject.cs

```diff
@@ -19,6 +19,11 @@ public class RavenSubject
         protected RavenSubject()
         {
             UniqueId = Guid.NewGuid();
+            var type = GetType();
+            if (RavenCache.RavenTypeWatcherCache.ContainsKey(type))
```

Comment:
faster is TryGetValue, but i'm not entirely sure if this is implemented in ConcurrentDictionary

Reply:
TryGetValue is faster than plain ContainsKey? thats strange, i think you maybe mean that TryGetValue is faster than ContainsKey and then dict[key] correct? 

If i am correct then i changed implementation where it occures. Thx - but let me know if i got it wrong